src/mngt/ocf_mngt_cache.c

```diff
@@ -2561,7 +2654,7 @@ void _ocf_mngt_cache_failover_detach(ocf_cache_t cache,
 }
 
 
-static void _ocf_mngt_cache_activate(ocf_cache_t cache,
+static void _ocf_mngt_cache_standby_activate(ocf_cache_t cache,
 		struct ocf_mngt_cache_device_config *device_cfg,
 		_ocf_mngt_cache_attach_end_t cmpl, void *priv1, void *priv2)
```

Comment:
Please add "open_cores" parameter, similarly to cache load - this improves testability in pyocf

Reply:
Done.